docs/user/features/extdep.md

```diff
@@ -265,7 +265,7 @@ recommended to use this to ensure the contents are as expected.
 ## Git Type Schema Differences
 
 - source: url of git repo
-- version: commit hash to checkout
+- version: commit hash to checkout (short or long)
```

Comment:
Though 7 digits is generally accepted as the default short hash length, I think it would be helpful to explicitly say something like "first 7 digits".

Reply:
Added a note that a short commit is considered a the first 7 digits of a commit hash.